### PR TITLE
Increase default money withdrawal amount

### PR DESCRIPTION
**Pull request by dlsf**
10 is **very** little money

### CODE REVIEW

**Comment by PeachesMLG** [comment]
I think this should be 1k

**Review by PeachesMLG — APPROVED** [review]
changed to 1k

**Review by PeachesMLG — CHANGES_REQUESTED** [review]
changed to 1k

**Comment by PeachesMLG** [comment]
Well, I tried editing this myself, but it wouldnt let me commit :cry:

File: src/main/java/com/iridium/iridiumskyblock/configs/BankItems.java [hunk 1]
```diff
@@ -18,6 +18,6 @@
 
     public CrystalsBankItem crystalsBankItem = new CrystalsBankItem(10, new Item(XMaterial.NETHER_STAR, 13, 1, "&b&lIsland Crystals", Arrays.asList("&7%amount% Crystals", "&b&l[!] &bLeft click to withdraw", "&b&l[!] &bRight click to deposit")));
     public ExperienceBankItem experienceBankItem = new ExperienceBankItem(10, new Item(XMaterial.EXPERIENCE_BOTTLE, 15, 1, "&b&lIsland Experience", Arrays.asList("&7%amount% Experience", "&b&l[!] &bLeft click to withdraw", "&b&l[!] &bRight click to deposit")));
-    public MoneyBankItem moneyBankItem = new MoneyBankItem(10, new Item(XMaterial.PAPER, 11, 1, "&b&lIsland Money", Arrays.asList("&7$%amount%", "&b&l[!] &bLeft click to withdraw", "&b&l[!] &bRight click to deposit")));
+    public MoneyBankItem moneyBankItem = new MoneyBankItem(100, new Item(XMaterial.PAPER, 11, 1, "&b&lIsland Money", Arrays.asList("&7$%amount%", "&b&l[!] &bLeft click to withdraw", "&b&l[!] &bRight click to deposit")));
```
Comment (dlsf) [inline src/main/java/com/iridium/iridiumskyblock/configs/BankItems.java]:
```suggestion
    public MoneyBankItem moneyBankItem = new MoneyBankItem(1000, new Item(XMaterial.PAPER, 11, 1, "&b&lIsland Money", Arrays.asList("&7$%amount%", "&b&l[!] &bLeft click to withdraw", "&b&l[!] &bRight click to deposit")));
```

**Comment by dlsf** [comment]
If nothing works, just suggest the change and commit it yourself

**Comment by PeachesMLG** [comment]
> If nothing works, just suggest the change and commit it yourself

I tried lol, too dum ig

**Pull request merged**